templates/site/js/load-node-content.js

```diff
@@ -29,10 +29,27 @@ function buildNodeDescription(name, data, node_index, row_id_suffix) {
                     img_alt = link.alt;
                 }
                 break
+
         }
     })
+
+    if(window.location.href.includes("ide.html")){
+        data.services.forEach(service => {
+            if(service.keywords.includes("jupyterhub")){
+
+                service.links.forEach(link => {
+                                if(link.href.includes("jupyter")){
+                                    node_template.innerHTML = node_template.innerHTML.replaceAll("{{href}}", link.href)
```

Comment:
why not just set the `href` variable here and then `break`. Then you don't need the `else` statement that you've added and you can keep the original `replaceAll` call that you've removed. 

This strategy adds a lot of unnecessary complexity to the code.